### PR TITLE
Cross build to Scala 2.13.0-RC2

### PR DESCRIPTION
**Pull request by ignasi35**
supersedes https://github.com/akka/akka-persistence-cassandra/pull/483 (can't push to @renatocaval's fork)

### CODE REVIEW

**Comment by TimMoore** [comment]
So it looks like we can keep publishing 2.11 artifacts for this stable branch, as well as 2.12 and 2.13, right? I guess the typed tests were added after branching.

This is good. We might still need to update APC in Lagom 1.4/1.5 for as long as they are supported, if there are bugs that need to be fixed. If the 0.6x series were to drop Scala 2.11 support, it would complicate this. We might end up needing another branch. Looks promising that we can avoid this.

File: cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala [hunk 1]
```diff
@@ -107,7 +108,7 @@ object CassandraLauncher {
         case jarUrl if jarUrl.getProtocol == "jar" =>
           new File(URI.create(jarUrl.getPath.takeWhile(_ != '!'))).getCanonicalPath
       }
-    }.distinct.to[immutable.Seq]
+    }.distinct.to(immutable.Seq)
```
Comment (2m) [inline cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala]: This is the only place where the added `collectionCompat` is used. We could instead do `.distinct.toIndexedSeq` and the collectionCompat would not be needed.

Comment (ignasi35) [reply cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala]: I'll have a look

Comment (dwijnand) [reply cassandra-launcher/src/main/scala/akka/persistence/cassandra/testkit/CassandraLauncher.scala]: TIL `IndexedSeq` in Scala 2.12 is `scala.collection.IndexedSeq` but `toIndexedSeq` returns `scala.collection.immutable.IndexedSeq`.

File: .travis.yml [hunk 2]
```diff
@@ -3,16 +3,32 @@ sudo: false
 scala:
   - "2.11.12"
   - "2.12.8"
+  - "2.13.0-RC2"
 jdk:
   - oraclejdk8
   - openjdk11
```
Comment (2m) [inline .travis.yml]: Build matrix does not compose nicely with stages, but that can be improved later.

Comment (ignasi35) [reply .travis.yml]: Yeah, I noticed duplicate jobs while working on #484 and raised https://github.com/akka/akka-persistence-cassandra/issues/485

**Comment by dwijnand** [comment]
Is someone cutting a release of this for Lagom?

**Comment by TimMoore** [comment]
@dwijnand 0.62 was released http://repo1.maven.org/maven2/com/typesafe/akka/akka-persistence-cassandra_2.13.0-RC2/0.62/

**Pull request merged**